hackerrank/src/GetDay.java

```diff
@@ -0,0 +1,65 @@
+import java.io.*;
+import java.util.Calendar;
+import java.util.Scanner;
+
+class FindDay {
+    public static String findDay(int month, int day, int year) {
+        /*Scanner sc = new Scanner(System.in);
+        month = sc.nextInt();
+        day = sc.nextInt();
+        year = sc.nextInt();*/
+        Calendar cal = Calendar.getInstance();

```

Comment:
Did you get notification for this comment?

Reply:
yes